client/src/app/ui/modules/datepicker/components/daterangepicker/daterangepicker.component.html

```diff
@@ -2,8 +2,8 @@
     <mat-form-field class="datepicker-form-field">
         <mat-label *ngIf="title">{{ title | translate }}</mat-label>
         <mat-date-range-input [formGroup]="contentForm" (click)="picker.open()" [rangePicker]="picker">
-            <input matStartDate readonly formControlName="start" placeholder="Start date" />
-            <input matEndDate readonly formControlName="end" placeholder="End date" />
+            <input matStartDate readonly formControlName="start" placeholder="{{'Start date' | translate}}" />
+            <input matEndDate readonly formControlName="end" placeholder="{{'End date' | translate}}" />
```

Comment:
I would prefer it like this
```suggestion
            <input matStartDate readonly formControlName="start" [placeholder]="'Start date' | translate" />
            <input matEndDate readonly formControlName="end" [placeholder]="'End date' | translate" />
```